jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -372,6 +377,24 @@ cassandraBranches.each {
         }
     }
 
+    /**
+     * Main branch test script target jobs
+     */
+    testScriptTargets.each {
+        def targetName = it
+
+         job("${jobNamePrefix}-${targetName}") {
```

Comment:
can we, for now, remove the `testScriptTargets` array and the each-loop here? ie, just make this a one-off `job(…)` like "*-artifacts",  "*-cqlsh-tests", and the pipelines.
(or did you have further jobs in mind to add to this array?)

Reply:
done.